Sources/Orbit/Components/Heading.swift

```diff
@@ -22,7 +22,7 @@ public struct Heading: View {
         }
     }
 
-    func textContent(sizeCategory: ContentSizeCategory) -> SwiftUI.Text {
+    @ViewBuilder func textContent(sizeCategory: ContentSizeCategory) -> SwiftUI.Text {
```

Comment:
What's the point of `@ViewBuilder` if we don't return `some View`?

Reply:
Updated